docs/user/openstack/README.md

```diff
@@ -49,6 +49,94 @@ hang waiting for the console to come up.
 `DEBUG Still waiting for the console route: the server is currently unable to
 handle the request (get routes.route.openshift.io)`
 
+## Using an External Load Balancer
+
+This documents how to shift from the api VM load balancer (which is not
+HA) to an external load balancer.
+
+The load balancer must serve ports 6443, 443, and 80 to any users of
+the system.  Port 49500 is for serving ignition startup configurations
+to the OpenShift nodes and should not be reachable by the outside world.
+
+The first step is to add floating IPs to all the master nodes. Usually
+the public network here is named 'public':
+
+* `openstack floating ip create --port master-port-0 <public network>`
+* `openstack floating ip create --port master-port-1 <public network>`
+* `openstack floating ip create --port master-port-2 <public network>`
+
+Once complete you can see your floating IPs using:
+
+* `openstack server list`
+
+These floating IPs can then be used by the load balancer to access
+the cluster.  An example haproxy configuration for port 6443 is below.
+The other port configurations are identical.
+
+```
+listen <cluster name>-api-6443
+    bind 0.0.0.0:6443
+    mode tcp
+    stats enable
```

Comment:
Stats are not required so it'd be better to remove them from this example config.